src/app/components/Grid/index.jsx

```diff
@@ -14,6 +14,7 @@ import {
 import {
   GEL_MARGIN_BELOW_400PX,
   GEL_MARGIN_ABOVE_400PX,
+  GEL_SPACING_HLF_TRPL,
```

Comment:
Has this amount been approved by UX?

Reply:
Great observation @HarryVerhoef. This is supposed to be `1rem` or `GEL_SPACING_DBL`